doc/dev/developer_guide/dash-devel.rst

```diff
@@ -2267,6 +2267,46 @@ Usage example:
   }
 
 
+How to create an issue in the Ceph Issue Tracker
```

Comment:
IMHO this documentation should be placed in `doc/mgr/dashboard.rst` as this feature is available to all users, not only developers.